lib/WTSI/NPG/HTS/Annotator.pm

```diff
@@ -11,7 +11,7 @@ our $VERSION = '';
 
 our @COMPRESSION_SUFFIXES     = qw[bz2 gz xz zip];
 
-our @GENERAL_PURPOSE_SUFFIXES = qw[bin csv h5 tar tgz tif tsv txt xls xlsx xml];
+our @GENERAL_PURPOSE_SUFFIXES = qw[bin csv h5 tar tgz tif tsv txt xls xlsx xml pbi];
```

Comment:
Ah old.  Thanks, I'll get rid of this as it was already added to perl-irods-wrap in https://github.com/wtsi-npg/perl-irods-wrap/pull/123